src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -159,9 +158,16 @@ class WatershedFilterAlgo : public ITKFilter
 	double watershedLevel_val_;
 
 	std::vector<std::string> dst_layer_ids_;
-	std::vector<_LabelType> watershed_image_output_labels_;
-  
+	std::vector<WtrLabelType> watershed_image_output_labels_;
+
 public:
+
+	void report_debug_message( const std::string& msg )
```

Comment:
This is redundant, output from CORE_LOG_DEBUG already reports the class. For example, a typical debug build logging message is: [2016-Jun-29 17:42:59|ActionWatershedFilter.cc|168] DEBUG: Watershed Filter: Initializing watershedding filter...